lib/customer.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'restaurant'
+require_relative 'message_sender'
+
+class Customer
```

Comment:
This class is largely redundant as all it does is delegate method calls to Restaurant.  It doesn't add any additional layers of abstraction or functionality